src/main/java/org/datadog/jmxfetch/reporter/StatsdReporter.java

```diff
@@ -35,13 +35,18 @@ public StatsdReporter(String statsdHost, int statsdPort) {
 
     private void init() {
         initializationTime = System.currentTimeMillis();
+        /* Create the StatsDClient with "entity-id" set to "none" to avoid
+           having dogstatsd server adding origin tags, when the connection is
+           done with UDS. */
         statsDClient =
                 new NonBlockingStatsDClient(
                         null,
                         this.statsdHost,
                         this.statsdPort,
+                        Integer.MAX_VALUE,
                         new String[] {},
-                        new LoggingErrorHandler());
+                        new LoggingErrorHandler(),
+                        "none");
```

Comment:
shouldn't we only be doing this if `this.statsdPort == 0`, otherwise this will also apply to UDP connections.

Reply:
good catch! I'm updating the PR